taskfiles/remote/tests.yaml

```diff
@@ -4,21 +4,38 @@ includes:
   remote: "../../exports/taskfiles/utils/remote.yaml"
 
 vars:
-  G_EXTRACTED_ZIP_CODEOWNERS_PATH: "yscope-dev-utils-main/.github/CODEOWNERS"
-  G_EXTRACTED_ZIP_LICENSE_PATH: "yscope-dev-utils-main/LICENSE"
+  G_EXTRACTED_ROOT_DIR: "yscope-dev-utils-fd7c42dd7b59f8f4ab0eccba5078393e10cddb00"
```

Comment:
Can we extract the commit hash into a variable so that we can use it here and in `G_TEST_ZIP_FILE_URL`? (GitHub allows using the complete commit hash to download a zip file.